core/consensus/common/src/import_queue.rs

```diff
@@ -400,6 +400,9 @@ impl<B: BlockT> BlockImporter<B> {
 		let success = self.justification_import.as_ref().map(|justification_import| {
 			justification_import.import_justification(hash, number, justification).is_ok()
 		}).unwrap_or(false);
+		if !success {
+			debug!("Justification import failed for hash: {:?} number: {:?} coming from node: {:?}", hash, number, who);
```

Comment:
Can we move this to inside the `map` above so we can log the error itself?

Reply:
yes indeed